.github/PULL_REQUEST_TEMPLATE/new_entropy_template.md

```diff
@@ -0,0 +1,45 @@
+## Contributor guidelines
+
+- [ ] I verify that I have read the
+    [devdocs](https://juliadynamics.github.io/Entropies.jl/dev/devdocs/).
+
+## Description and references
+
+What is this new entropy called? Please cite appropriate scientific
+literature. Outline any implementation details we should be aware of.
+
+## What type of entropy (estimator) is this?
+
+- [ ] Computes entropy directly from probabilities that are explicitly estimated.
+- [ ] Computes entropy indirectly, in way that doesn't explicitly estimate probabilities.
+
+## Implementation
+
+- [ ] The new entropy is a subtype of `Entropy` or `IndirectEntropy`.
+- [ ] Dispatch for `entropy` is implemented.
+- [ ] Dispatch for `entropy_maximum` is implemented. If not, why not?
+
+## Documentation
+
+- [ ] The new entropy type has a docstring.
+- [ ] The docstring shows, using LaTeX notation, the entropy formula.
+- [ ] The docstring references original scientific literature, if relevant.
+- [ ] A level-two subheading, titled `MyNewEntropy` is included in
+    `docs/src/entropies.md`, and this section contains a `@docs` block
+    referencing `MyNewEntropy`.
+- [] A runnable example is included in the `docs/src/example.md` file.
+
+## Testing
+
+- [ ] Tests are implemented.
+- [ ] Test cases with known input/output values which can be tested exactly are included.
+    If not, why didn't you include such tests?
+
+## Checklist before requesting a review
+
+I confirmed that I have
+
+- [ ] Self-reviewed my code.
```

Comment:
I would remove both the "I confirm" and the self revieew.

Reply:
> We have to discuss this a bit more. I am not particularly happy with the way they are right now, because I don't perceive a tone that reflects how JuliaDynamics operates with new contributors. (I typically give people push access after two PRs or so). I want to be sure that if indeed we have such PR guidelines, they reflect a warm, friendly, and welcoming atmosphere. Something that has long checklists of 15+ points, and requiring the contributor to "explicitly declare things", is rather formal and may even come across as intimidating.

> Besides this, we should also consider whether we need checklists at all. Why aren't the devdocs enough? We could just add one more devdoc method for entropy methods. The crucial difference with the devdoc is that it is about exposition of infirmation. It is passive. Here it is different. The author has already taken the decision to be involved, and they need to "declare things" and follow a 15-check-long checlist. From passive guidelines it became an active critique of the PR.

The reason why I phrased the templates in this particular (explicit) manner is from my teaching experience. By giving  explicit expectations regarding quality, we've seen immense increase in quality of student creations. This is also the case for myself: by forcing myself to explicitly write down a list of things I've done for each PR I've submitted, I also immediately notice if something is missing.

Although I don't quite see how a set of established standards, in itself, would come off as neither intimidating nor non-inclusive. However, we can easily change the language and shorten the number of check boxes and relax the formal tone, if that would feel more welcoming to more people.

I think we should keep, as a minimum, a few checkboxes relating to
- [ ] Tests are included and pass.
- [ ] Documentation (types are referenced in the docs, and  possibly examples are provided)

We can skip the checkbox entirely for "verify that you've read the devdocs". 

I'll shorten things a bit, and rephrase, then we can re-discuss.

Reply:
Ok, I've made a shorten, less formal suggestion below. What do you think, @Datseris?